### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by thaJeztah**
because this repo wouldn't be complete without a nit fixed :D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ $ ./bin/alpine -h
 ## Caveats
 
 **Caps the binary needs to unpack and set 
-the right perms on the roofs for the userns user**
```
Comment (vbatts) [inline README.md]: but ... the roofs on fire. We don't need no water ...

**Comment by thaJeztah** [comment]
Lol, looks like it isn't my day.. hah, don't know what I did, got some of your commits in here

File: README.md (hunk 1 above)
Comment (jfrazelle) [inline README.md]: y'all are crazy

Comment (jfrazelle) [inline README.md]: BUT I <3 SO MUCH

Comment (thaJeztah) [inline README.md]: Group hug!!

**Pull request closed without merging**